OrchardCore.AuditTrail/Services/AuditTrailManager.cs

```diff
@@ -103,7 +103,7 @@ public async Task AddAuditTrailEventAsync<TAuditTrailEventProvider>(AuditTrailCo
             Filters filters = null,
             AuditTrailOrderBy orderBy = AuditTrailOrderBy.DateDescending)
         {
-            var session = _session.Store.CreateSession(System.Data.IsolationLevel.ReadUncommitted);
+            var session = _session.Store.CreateSession();
```

Comment:
Does this change matter?

Reply:
Ideally this module shouldn't be used at the moment. I can only imagine it was added to the solution in order to be evaluted. This has been superceded by a much more integrated current version.